xdrip-get-entries.sh

```diff
@@ -8,9 +8,43 @@ main()
 
 # Cmd line args - transmitter $1 is 6 character tx serial number
   transmitter=$1
-  cmd_line_mode=${2:-""}
-  pumpUnits=${3:-"mg/dl"}
-  meterid=${4:-"000000"}
+  if [ -z  "$transmitter" ] || [ "$transmitter" == "-" ]; then
```

Comment:
I've tested and this works for me. Just wondering why you added the or = "-"

Reply:
I'm wondering as well.  At one point I may have been thinking something funny and then forgot to clean it up.  I'll look at removing the extra condition to clean up the code and avoid confusion. 
Good catch.